doc/guides/onboarding-extras.md

```diff
@@ -20,6 +20,8 @@ request.
 * `discuss` - Things that need larger discussion
 * `feature request` - Any issue that requests a new feature (usually not PRs)
 * `good first issue` - Issues suitable for newcomers to process
+* `wait-for-feedback` - Keep this pull request open at least 48 hours before
```

Comment:
The labels are in alphabetical/lexical order right now. I'd prefer we keep it that way and move this one to the end.